src/proto/batch/mod.rs

```diff
@@ -235,6 +239,34 @@ impl<'a, S: Read + Write> BatchHandle<'a, S> {
     }
 }
 
+// Not documented, but existing de fakto and also mentioned in the official client
+// https://github.com/contribsys/faktory/blob/main/client/batch.go#L17-L19
+/// State of a `callback` job of a [`Batch`].
+#[derive(Debug, Clone, Deserialize)]
```

Comment:
Let's add `Copy`, `PartialEq`, and `Eq`.

Reply:
Also added them for `JobState`